dev/EnvironmentManager/ChangeTrackerHelper/ChangeTrackerHelper/ChangeTrackerHelper.h

```diff
@@ -0,0 +1,56 @@
+﻿#pragma once
+#include <windows.h>
+#include <appmodel.h>
+
+enum AreChangesTracked
+{
+    YES,
+    NO,
+    NOT_CHECKED
+};
```

Comment:
The classic Schroedinger Boolean -- Yes, No, DontKnowYet ;-)

Isn't the canonical C++ solution to use `std::optional<bool>`?